docs/data/xrootd/install-standalone.md

```diff
@@ -127,6 +127,33 @@ To set up XRootD in multi-user mode, install the `xrootd-multiuser` package:
 root@xrootd-standalone # yum install xrootd-multiuser
 ```
 
+#### Throttling IO requests
+
+XRootD allows throttling of requests to the underlying filesystem.
+To enable this,
+
+1.  In an `/etc/xrootd/config.d/*.cfg` file, e.g. `/etc/xrootd/config.d/99-local.cfg`, set the following configuration:
+
+        xrootd.fslib throttle default
+        throttle.throttle concurrency <CONCUR> data <RATE>
+
+    Replacing `<CONCUR>` with the IO concurrency limit and `<RATE>` with the data rate limit in bytes.
```

Comment:
What are the units for CONCUR? For RATE, you should have "bytes per second", not "bytes". Is there no way to specify it in KB/s or MB/s?

What are some good numbers for these? Can you find some examples?

Reply:
Updated the explanation. I don't know of any examples, though.